src/Homepage.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactElement } from "react";
+import SearchIcon from "./assets/SearchIcon.svg";
+import Alert from "./components/Alert";
+import EventCard from "./components/EventCard";
+import EventsJSON from "./events.json";
+
+const Homepage = (): ReactElement => {
+  function getColor(start: Date, end: Date) {
+    const today = new Date();
+    const timeNow = today.getTime();
+
+    if (start.getTime() < timeNow && timeNow < end.getTime()) {
+      return "bg-sga-red";
+    } else if (today.toDateString() === start.toDateString()) {
+      return "bg-black";
+    } else {
+      return "bg-white";
+    }
+  }
+
+  const liveEvents = EventsJSON.filter(
+    (e) => getColor(new Date(e.startTime), new Date(e.endTime)) === "bg-sga-red"
```

Comment:
nit: I wonder if it would be helpful to actually convert EventsJSON into a list of Events first, so that we don't have to keep doing `new Date(e.startTime)`